api/v1/utils/logs.js

```diff
@@ -25,25 +23,42 @@ function _filterBody(body, blacklist) {
   return _.omit(body, blacklist);
 }
 
-function _makeRequestMetadata(req) {
+function _acknowledgeRequest(req) {
   return {
     id: req.id,
     method: req.method,
-    url: req.originalUrl
+    url: req.originalUrl,
+    ip: req.ip
+  }
+}
+
+function _makeRequestMetadata(req) {
+  return {
+    id: req.id,
+    user: req.user ? req.user.get('id') : 'Unauthenticated',
+    adminOverRide: req.originUser ? req.originUser : 'No impersonation',
+    query: req.query,
+    params: req.params,
+    body: _filterBody(req.body, REQUEST_BLACKLIST)
   };
 }
 
 function _makeResponseMetadata(req, res) {
   return {
     id: req.id,
-    status: res.statusCode
+    status: res.statusCode,
+    body : res.body
   };
 }
 
 module.exports.errorTypes = ERROR_TYPES;
 
+module.exports.logRequestReceipt = (req) => {
+  logger.debug('received request', _acknowledgeRequest(req));
+}
+
 module.exports.logRequest = (req) => {
-  logger.debug('received request', _makeRequestMetadata(req));
+  logger.debug('request finalized', _makeRequestMetadata(req));
```

Comment:
I wouldn't normally care about a debug line, but considering this is going to be in the logs so frequently I thought I'd suggest a change. To keep it consistent with the `logRequestReceipt` log line, maybe this should be 'finalized request' as opposed to the other way around.

I'm also wondering if a different word than 'finalized' might be better understood. I know what you're getting at, but it feels sort of unclear to say it's finished when it's only passed through the middleware. What about something like 'certified request' or 'qualified request'? Feel free to use really anything else or to ignore this paragraph, but I thought it was worth mentioning.